modules/adding-bare-metal-host-to-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="adding-bare-metal-host-to-cluster-using-web-console_{context}"]
+= Adding a bare metal host to the cluster using the web console
+
+You can add bare metal hosts to the cluster in the web console.
+
+.Prerequisites
+
+* Install an {op-system} cluster on bare metal.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the web console, navigate to *Compute* -> *Bare Metal Hosts*.
+. Select *Add Host* -> *New with Dialog*.
+. Specify a unique name for the new bare metal host.
+. Set the boot MAC address.
```

Comment:
it would be better if we use `Boot MAC Address` with bold text

<img width="668" alt="Screen Shot 2021-06-16 at 10 11 41 AM" src="https://user-images.githubusercontent.com/12692381/122147508-9f1a2980-ce8b-11eb-85fa-0257952b9f3c.png">